salesagility/SuiteCRM
# Fixed - Text wrapping issue on detailview/editview

## Description



Fixed an issue with text wrapping on editview/detailview. Before fix:

![case_text_issue](https://user-images.githubusercontent.com/26431166/27635694-fef49e74-5bff-11e7-8b63-fca7d9c0af0a.png)

After fix:

![case_text_issue_1](https://user-images.githubusercontent.com/26431166/27635728-27f84938-5c00-11e7-9721-e94a6d98a809.png)

## Motivation and Context

This allows the creation of fields that have a large number of characters. Previously, the label would be covered up by the following field.

## How To Test This

1. Delete cache
2. Create a custom field for a module in studio. 
3. Ensure that the custom field has a large label.
4. Add the field to a module.
5. Check that the label appears on the next line instead of under the next field. 

## Types of changes


### Final checklist
